lab_3/main.py

```diff
@@ -181,3 +197,33 @@ def split_by_sentence(text: str) -> list:
         element = element.split()
         my_tokens.append(element)
     return my_tokens
+
+
+test_text = "Mar#y wa$nted, to swim! \n However, she  \n was afraid of sharks. However, she  \n afraid of dogs. She afraid of dogs."
+sentences = split_by_sentence(test_text)
+ws = WordStorage()
+for sentence in sentences:
+    ws.from_corpus(tuple(sentence))
+
+# print(sentences)
+ngram = NGramTrie(3)
+
+for sentence in sentences:
+    encoded = encode(ws, sentences)
+# print(encoded)
+
+for enc in encoded:
+    # print(enc)
+    ngram.fill_from_sentence(tuple(enc))
+
+ngram.calculate_log_probabilities()
+
+# print(ngram.gram_log_probabilities)
+# print(ngram.gram_frequencies)
+# print(word_storage.storage)
```

Comment:
@dmitry-uraev clean up your work - remove all prints and redundant comments

Reply:
done